converter/src/main/java/gov/cms/qpp/conversion/validate/MipsQualityMeasureIdValidator.java

```diff
@@ -12,6 +12,7 @@
 import gov.cms.qpp.conversion.model.validation.MeasureConfigs;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'gov.cms.qpp.conversion.model.validation.MeasureConfigs'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)